engine/rowset/src/main/java/io/deephaven/engine/rowset/impl/ExternalizableRowSetUtils.java

```diff
@@ -107,7 +107,6 @@ private static void flushShorts(@NotNull final DataOutput out, @NotNull final TS
                         consecutiveBytes++;
                     } else {
                         consecutiveBytes = 0;
-                        shortCount += consecutiveBytes;
```

Comment:
The very next line unconditionally sets shortCount, so this += is always ignored.

Reply:
That's not accurate. The next line is:
```
shortCount++;
```

Reply:
I'm sorry, I misremembered what the issue was, its been a few weeks.

The _real_ reason this is useless is the line prior, which unconditionally sets `consecutiveBytes` to zero, so this `+=` has no effect.

Reply:
Ah. Good point. Maybe the correct fix is to re-order those two lines?

Reply:
Tests don't fail by inverting those lines, which isn't my favorite thing...

Reviewing history in DHE, these lines have been ordered like this since 2015, which looks like its initial commit.

Reply:
Reverted this change for you to follow up on instead.